validator/lib/validateTileset.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var Promise = require('bluebird');
+
+module.exports = validateTileset;
+
+/**
+ * Checks whether the geometricError of each tile is <= the geometric error of its parent tile.
+ *
+ * @param {JSON} The JSON object representing the tileset.
```

Comment:
Change `{JSON}` to `{Object}`

Also the doc needs to include the parameter name:

`@param {Object} tileset The JSON object representing the tileset.`